CONTRIBUTING.md

```diff
@@ -0,0 +1,197 @@
+[installing-git]: https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
+[installing-npm]: https://nodejs.org/en/download/
+[pro-git]: https://git-scm.com/book/en/v2
+[conda-dot-org-repo]: https://github.com/conda-incubator/conda-dot-org
+[vscode-download]: https://code.visualstudio.com/?wt.mc_id=vscom_downloads
+[sublimetext-download]: https://www.sublimetext.com/3
+[atom-download]: https://atom.io/
+[docusaurus]: https://docusaurus.io/
+[markdown-guide]: https://www.markdownguide.org/
+[issue-board]: https://github.com/orgs/conda-incubator/projects/3
+
+# Contributing
+
+Welcome to the conda.org contributing guide! In this guide, you will find everything you need
+to get up and running as a contributor to conda.org. This is a contributing guide
+for two kinds of contributors:
+
+- **Content contributors**
+    - Contributors writing content for the website, which can include news and blog posts.
+- **Technical contributors**
+    - Contributors writing HTML, CSS, or Javascript for the website that modifies the way 
+      it looks or behaves.
+
+In this guide, we start off by explaining contribution steps that apply
+to both kinds of contributors. Then, the guide branches off to explain specifics for each.
+
+## Documentation tools
+
+conda.org is built with [Docusaurus 2][docusaurus], a modern static website generator that 
+uses a friendly Markdown format to create the website copy. If you are not familiar with 
+Markdown, check out the [Markdown Guide project][markdown-guide].
+
+## Working on issues
+
+The conda.org project has an [issue board][issue-board] from which technical contributors can choose issues. If there is an aspect of the website that you feel needs work, please create a new issue using the **Add item** button.
+
+To be assigned to an existing issue, comment on it and one of the maintainers will ensure you are assigned.
+
+## Github flow
+
+### Prerequisites
+
+To follow this guide, you will need to have these programs installed on your computer:
+
+- git: [installation link][installing-git]
+
+It is also highly encouraged that you at least have a basic understanding of how to use
+git at the command line. If you do not, please check out the book [Pro Git][pro-git],
+available for free from the official git website.
+
+### Github flow
+
+#### 1. Creating a fork
+
+The very first step to contributing is creating your own fork of the 
+[conda-dot-org repository][conda-dot-org-repo]. If you have never done this before, 
+we strongly encourage you to read the 
+[Fork a repo][github-forking-how-to] guide from GitHub first.
+
+#### 2. Clone your repository
+
+After you have created your fork, you are ready to clone this repository to your 
+local computer, where you can begin making edits. The commands for doing so will 
+be different depending on your user name, but a typical example is shown below:
+
+```
+$ git clone git@github.com:username/conda-dot-org.git
+```
+
```

Comment:
Somewhere in here should we say "Move to the newly cloned directory."

``` Bash
$ cd conda-dot-org
```

Reply:
Good idea to put this earlier. I also made it a bit less complex.